packages/story-editor/src/app/font/actions/useLoadFontFiles.js

```diff
@@ -19,12 +19,12 @@
  */
 import { useCallback } from '@googleforcreators/react';
 import { getGoogleFontURL } from '@googleforcreators/fonts';
+import { loadStylesheet } from '@googleforcreators/dom';
 
 /**
  * Internal dependencies
  */
 import cleanForSlug from '../../../utils/cleanForSlug';
-import loadStylesheet from '../../../utils/loadStylesheet';
 import { ensureFontLoaded, loadInlineStylesheet } from '../utils';
```

Comment:
`loadInlineStylesheet` could also be something for the `dom` package?

Reply:
Didn't want to add `fonts` package dependency to `dom`. But yes it can be removed by a little refactoring. Done in https://github.com/GoogleForCreators/web-stories-wp/pull/10717/commits/be17eff2a932febd7b1659a1c4a740e099507414